nmmo/core/env.py

```diff
@@ -133,6 +137,7 @@ def reset(self, map_id=None, seed=None, options=None):
     self._init_random(seed)
     self.realm.reset(map_id)
     self._dead_agents = OrderedSet()
+    self.packet_manager.reset()
```

Comment:
Does this need to be in Env? I would much rather it went into realm. I think of Env as dealing with the RL side of things, whereas rendering is part of the game environment.

Reply:
Env is for RL, realm for the game, thanks for the clarification! I'm moving "ReplayHelper" (renaming from PacketManager) to the realm.